### DISCUSSION TITLE
Configuration Discovery for apollo

### DISCUSSION THREAD

**Discussion by zjw6985559** [apache/skywalking#9967] [Q&A]
![image](https://user-images.githubusercontent.com/19279321/202105008-5bd44a98-c78a-4561-8e56-ede26fc66244.png)

**Comment by pg-yang** [comment]
It's the problem of apollo, rather than SkyWalking

**Reply by zjw6985559** [reply]
We have 177 services, which exceed the maximum length limit of apollo according to the way of cds configuration on skywalking official website. Please ask whether cds dynamic configuration supports global configuration？ Our configuration format is as follows:

configurations:

  channel-up-settle:

    agent.trace. ignore_ path: /actuator/**,/eureka/**,/configs/**,Redisson/**

    agent. sample_ n_ per_ 3_ secs: 500

  channel-up-qrcode:

    agent.trace. ignore_ path: /actuator/**,/eureka/**,/configs/**,Redisson/**

    agent. sample_ n_ per_ 3_ secs: 500

   eads:

    agent.trace. ignore_ path: /actuator/**,/eureka/**,/configs/**,Redisson/**

    agent. sample_ n_ per_ 3_ secs: 500

  acas-trans:

    agent.trace. ignore_ path: /actuator/**,/eureka/**,/configs/**,Redisson/**

    agent. sample_ n_ per_ 3_ secs: 500

**Reply by pg-yang** [reply]
If the doc doesn't introduce global configuration, it's unsupported.

Refer to  https://skywalking.apache.org/docs/skywalking-java/next/en/setup/service-agent/java-agent/configuration-discovery/

BTW, welcome to contribute to this feature.

**Reply by wu-sheng** [reply]
Configuration group is an extension concept to support service-centric configuration, proposed when implemented OpenAPI configurations. https://skywalking.apache.org/docs/main/next/en/setup/backend/dynamic-config/#group-configuration

If you want to contribute to DCS implementation on the OAP side to support this(also keep the current one). I think this would not be an issue anymore.

**Reply by wu-sheng** [reply]
Also, you should ask apollo for suggestions. After all, we have very little knowledge about that project. They may have some suggestions for this from a more professional perspective.
